src/client/cli/cmd/launch.cpp

```diff
@@ -204,14 +204,14 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
                                   "cpus", default_cpu_cores);
     QCommandLineOption diskOption(
         {"d", "disk"},
-        QString::fromStdString(fmt::format("Disk space to allocate. Positive integers, in "
+        QString::fromStdString(fmt::format("Disk space to allocate. Positive decimals, in "
                                            "bytes, or with K, M, G suffix.\nMinimum: {}, default: {}.",
```

Comment:
How about

```suggestion
        QString::fromStdString(fmt::format("Disk space to allocate. Positive integers, in "
                                           "bytes, or decimals, with K, M, G suffix.\nMinimum: {}, default: {}.",
```

Reply:
That would be more specific and possibly avoid some future confusion :)